core/gear_log/file_handle.ex

```diff
@@ -77,6 +77,14 @@ defmodule AntikytheraCore.GearLog.FileHandle do
       " [" <> Atom.to_string(level) <> "] context=" <> context_id <> " "
   end
 
+  defun set_write_to_terminal(
+          {file_path, last_checked_at, io_device, _} = _handle :: t,
+          new_val :: nil | boolean \\ nil
+        ) :: t do
+    write_to_terminal? = if new_val != nil, do: new_val, else: determine_write_to_terminal()
```

Comment:
My favorite way is creating `restore_write_to_terminal/0` instead of using `nil`. IMHO, It's easier to understand than using `nil`.

Reply:
Indeed. 
I changed to use dedicated API (I think `FileHandle.restore_write_to_terminal` needs to take a `file_handle` as an argument, but is there another way to define as `restore_write_to_terminal/0`...?). (https://github.com/access-company/antikythera/pull/182/commits/bee7b442b5ebaa1f181d7e5c01e4f2eae3d41876)

Reply:
Sorry. `restore_write_to_terminal/1` is correct. I forgot `FileHandle.t`